app/src/main/java/de/westnordost/streetcomplete/tangram/QuestsMapFragment.java

```diff
@@ -193,7 +206,7 @@ private float getMaxZoomThatContains(ElementGeometry geometry)
 		BoundingBox screenArea;
 		float currentZoom;
 		synchronized(controller) {
-			screenArea = getDisplayedArea(questTopOffset, questBottomOffset);
+			screenArea = getDisplayedArea(questOffset.top, questOffset.bottom);
```

Comment:
I fear you need to adapt this method as well to take a `Rect`.

Reply:
But it's relatively straightforward.

Reply:
I have no idea at the moment how I could do this. Could you help me?

Reply:
Please try to understand the code. If you give up so quickly, I might as well write the code myself.